peachjam/models/judgment.py

```diff
@@ -339,6 +354,14 @@ def pre_save(self):
         if self.registry:
             self.court = self.registry.court
 
+        # ensure judgment locality matches the court's locality
+        if self.locality and self.court.locality:
```

Comment:
surely we want the locality to be set if the court has one, even if the self.locality is unset? Same for jurisdiction. If the court has either, set it on the judgment.